node/primitives/bridge/Cargo.toml

```diff
@@ -11,6 +11,7 @@ version     = "2.7.2"
 
 [dependencies]
 # darwinia-network
+common-runtime         = { default-features = false, path = "../../runtime/common" }
```

Comment:
Primitives should not depend on runtime.

Reply:
if so, we need define the types independently

Reply:
How about creating an issue firstly, we can find a way to fix it later. So we can merge this pr today.

Reply:
I find the next version of parity-bridges-common. This definitions are changed too?